server/packet_handlers.py

```diff
@@ -396,6 +401,109 @@ async def send_private_message_handler(session: "Session", packet_data: bytes):
 # CREATE_MATCH = 31
 
 
+@bancho_handler(packets.ClientPackets.CREATE_MATCH)
+async def create_match_handler(session: "Session", packet_data: bytes):
+    own_presence = session["presence"]
+
+    if not own_presence["privileges"] & ServerPrivileges.UNRESTRICTED:
+        return
+
+    packet_reader = packets.PacketReader(packet_data)
+
+    osu_match_data = packet_reader.read_osu_match()
```

Comment:
here's an example usage of the functions i created in packets.py to simplify some things